onnxscript/rewriter/ort_fusions/_test_utils.py

```diff
@@ -27,13 +27,13 @@ def _save(model, modelpath):
 
 def ort_run(model_name: str, model, inputs):
     providers = ["CPUExecutionProvider"]
-    with tempfile.TemporaryDirectory() as temp_dir:
-        model_path = os.path.join(temp_dir, f"{model_name}.onnx")
-        _save(model, model_path)
-        # Run model
-        session = onnxruntime.InferenceSession(model_path, providers=providers)
-        ort_outputs = session.run(None, inputs)
-    return ort_outputs
+    onx = ir.serde.serialize_model(model)
+    opts = onnxruntime.SessionOptions()
+    opts.graph_optimization_level = (
+        onnxruntime.GraphOptimizationLevel.ORT_DISABLE_ALL
+    )
```

Comment:
Sometimes disabling all optimizations may make a valid model not runnable. Not sure why that happens but it seems like this optimization level was not thoroughly tested

Reply:
It is a bug but it should be rare on the examples ran through this function.